src/Progress/Progress.tsx

```diff
@@ -15,7 +15,7 @@ export interface ProgressProps extends HTMLAttributes<HTMLElement> {
   /** Classi aggiuntive da usare per il componente interno del Progress */
   className?: string;
   /** Valore corrente (numerico) */
-  value: number | string;
+  value?: number | string;
```

Comment:
This change has to be combined with the check below.
Please add an additional condition before logging an Error for the `indeterminate` prop.

Reply:
Added the falsy indeterminate in the Error log condition.